src/clients/SpokePoolClient.ts

```diff
@@ -570,7 +581,7 @@ export class SpokePoolClient extends BaseAbstractClient {
         deposit.fromLiteChain = this.isOriginLiteChain(deposit);
         deposit.toLiteChain = this.isDestinationLiteChain(deposit);
 
-        if (deposit.outputToken === ZERO_ADDRESS) {
+        if (deposit.outputToken === ZERO_ADDRESS || deposit.outputToken === ZERO_BYTES) {
           deposit.outputToken = this.getDestinationTokenForDeposit(deposit);
```

Comment:
This has some assumptions about how tokens are going to be stored in the hub pool client, namely, whether they are going to be stored all as bytes32 or as a mix of bytes32s and addresses. I'd probably prefer the latter.

Reply:
The HubPool is constrained to work with the `address` type, so I believe these will be "virtual" addresses, and then we would map those to non-EVM addresses separately.